docs/USAGE.md

```diff
@@ -0,0 +1,282 @@
+# Usage
+
+**WARNING**: Please notice that no official release has been created for this repo yet, so installation instructions below would require you to build the code manually beforehand.
+
+## Configure the OpenTelemetry Tracing Library for .NET
+
+### Configuration values
+
+Use these environment variables to configure the tracing library:
+
+| Environment variable | Default value | Description |
+|-|-|-|
+| `OTEL_SERVICE_NAME` |  | The name of the service. |
+| `OTEL_ENV` |  | The value for the `environment` tag added to every span. This determines the environment in which the service is available in OTEL µAPM.  |
```

Comment:
IIRC `environment` is an OpenTracing convention, we could actually start to use this one https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/deployment_environment.md

```suggestion
| `OTEL_ENV` |  | The value for the `environment` tag added to every span. |
```